src/cargo/util/toml/targets.rs

```diff
@@ -10,21 +10,24 @@
 //! It is a bit tricky because we need match explicit information from `Cargo.toml`
 //! with implicit info in directory layout.
 
-use std::path::{Path, PathBuf};
-use std::fs::{self, DirEntry};
 use std::collections::HashSet;
+use std::fs::{self, DirEntry};
+use std::path::{Path, PathBuf};
```

Comment:
This seems like an unrelated formatting change. (It's a good change, but unrelated to this commit.)